src/test/java/org/opensearch/security/ssl/SecureSSLSettingsTest.java

```diff
@@ -7,46 +7,11 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import org.opensearch.common.settings.MockSecureSettings;
-import org.opensearch.common.settings.Settings;
-
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.is;
-import static org.opensearch.security.ssl.SecureSSLSettings.SSLSetting.SECURITY_SSL_HTTP_PEMKEY_PASSWORD;
-
 public class SecureSSLSettingsTest {
```

Comment:
Should this class be renamed. It seems to only have 1 test now. It doesn't make sense to have this test class file.